static/styles/_base/_tables.scss

```diff
@@ -159,17 +165,31 @@ td {
     border-bottom: 1px solid $light-gray;
   }
 
+  .table {
+    &:last-child {
+      margin-left: 2%;
+    }
+  }
+
+  .table__row--header {
+    border-bottom: 0;
+    margin-left: -.75em;
+  }
+
 }
 
 .totals-table {
   @include font-size(1.6);
 
   .table__cell {
-    width: 40%;
+    width: 20%;
```

Comment:
I think we need different styles for the `.totals-table` on the candidate page and the one that's on the single committee financial summary because the width's are making this happen:

![screen shot 2015-04-21 at 11 43 40 am](https://cloud.githubusercontent.com/assets/1696495/7260014/f115eb8c-e81b-11e4-9e3b-91418c778ab5.png)